src/components/Button/index.tsx

```diff
@@ -55,9 +55,11 @@ const Button = React.forwardRef<HTMLButtonElement, Props>(function Button(
 ) {
     const className = classNames(
         'reactist_button',
```

Comment:
Note that we add `reactist_button` and `reactist_button--primary` classes etc. also in addition to CSS module classes to support Twist backwards-compatibility for now.

Reply:
But I insist: the css module classes should not (need not) be named in the same way the global classes were named.

Reply:
I guess it's okay for now but shortly after shipping themes (or by shipping them?) we should bump a major and remove this 👍